cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -28,7 +28,7 @@ THE SOFTWARE.
     <d:taglib uri="local">
         <d:tag name="listSteps">
             <j:forEach var="stepDescriptor" items="${stepDescriptors}">
-                <f:dropdownListBlock title="${stepDescriptor.displayName}" staplerClass="${stepDescriptor.clazz.name}" lazy="stepDescriptor,it">
+                <f:dropdownListBlock title="${stepDescriptor.functionName}: ${stepDescriptor.displayName}" staplerClass="${stepDescriptor.clazz.name}" lazy="stepDescriptor,it">
```

Comment:
`Snippetizer.getStepDescriptors` will need to be modified to sort by `functionName`. (`ExtensionComponent.compareTo` sorts them in most cases by `displayName`, and `StepDescriptor.all` and thus currently `Snippetizer.getStepDescriptors` simply inherits that order.)